src/objectTranslations/Contact-ca.objectTranslation

```diff
@@ -4422,6 +4427,11 @@
             <translation>Feina</translation>
         </picklistValues>
     </fields>
+    <fields>
```

Comment:
@lifewithryan Unfortunately, we have to make sure we only include Chosen Full Name on all of these because the Primary fields are unmanaged.

Reply:
DOH! Thanks...